theforeman.org/yaml/jobs/foreman-infra-jenkins-jobs-update.yml

```diff
@@ -4,7 +4,7 @@
     sandbox: true
     properties:
       - github:
-          url: https://github.com/theforeman/foreman-infra
+          url: https://github.com/theforeman/jenkins-jobs
```

Comment:
It feels odd to keep this job named `foreman-infra` while the job has been changed to a different repo.

Reply:
The intent was not to keep it that way long term, I have been trying to do this in stages this whole time to reduce the amount of places we might break something.